ui/app/adapters/capabilities.js

```diff
@@ -22,12 +22,8 @@ export default ApplicationAdapter.extend({
     return [`${relativeNamespace}/${path.replace(/^\//, '')}`];
   },
 
-  findRecord(store, type, id) {
-    const paths = this.formatPaths(id);
-    return this.ajax(this.buildURL(type), 'POST', {
-      data: { paths },
-      namespace: sanitizePath(this.namespaceService.userRootNamespace),
-    }).catch((e) => {
+  async findRecord(store, type, id) {
```

Comment:
We probably still need to format the paths?

Reply:
ugh  dange merge resovler i said to KEEP the paths. good catch